108.md

```diff
@@ -0,0 +1,170 @@
+NIP-108
+======
+
+Lightning Gated Notes
+-------------------------------
+
+`draft` `optional` `author:coachchuckff` `author:excalibur_guild`
+
+This NIP defines three events for gating Notes behind lightning paywalls: 
+
+- Lightning-Gated Note ( `kind:55` ): This note allows you wrap any type of note behind a lightning gated paywall by encrypting the payload with a purchasable decrypt key.
+- Key Note ( `kind:56` ): This note encrypts the key for a given note, per user, using [NIP-04](https://github.com/nostr-protocol/nips/blob/master/04.md) between the corresponding gate creator's public key and the purchasers. It is linked to the gated note with the `g` tag.
```

Comment:
Is the key note optional?

Reply:
Technically, yes, the key note is optional but highly recommended. You can always store your own decrypt keys, or whatever client you are using could. However, I'd try to limit the reliance on 3rd parties as much as possible. Storing the key note on Nostr makes it much more likely you always have access to the gated note.

Reply:
It makes sense.